ACE/include/makeinclude/platform_g++_common.GNU

```diff
@@ -74,6 +74,20 @@ endif
 
 CXX_FULL_VERSION := $(shell $(CXX_FOR_VERSION_TEST) --version)
 
+# Minimum C++ level is now C++17, gcc until version 11 have an older version as default
+ifeq ($(findstring $(CXX_MAJOR_VERSION),7),$(CXX_MAJOR_VERSION))
+  c++std ?= c++17
+endif
+ifeq ($(findstring $(CXX_MAJOR_VERSION),8),$(CXX_MAJOR_VERSION))
+  c++std ?= c++17
+endif
+ifeq ($(findstring $(CXX_MAJOR_VERSION),9),$(CXX_MAJOR_VERSION))
+  c++std ?= c++17
+endif
+ifeq ($(findstring $(CXX_MAJOR_VERSION),10),$(CXX_MAJOR_VERSION))
```

Comment:
This looks like this would match versions 17, 18, 19, 27, etc. If this was run with those versions it would downgrade the standard being used if the user didn't set `c++std`.

Reply:
This matches the code we had in the past for C++11, will have a look at this before the next micro, feel free to open a PR